packages/odyssey-storybook/src/components/odyssey-labs/DataTable/DataTable.stories.tsx

```diff
@@ -200,6 +202,14 @@ const storybookMeta: Meta<DataTableMetaProps> = {
       control: "boolean",
       name: "[STORY ONLY] Has action buttons in table view?",
     },
+    hasAdditionalActionButton: {
```

Comment:
I feel like the API for this component is already pretty complex. Adding Story only props to the documentation feels like it could cause confusion maybe.

Reply:
I see that we've already done this but, feels like we should make an effort to remove them IMHO